MedicalDocManagment.WebUI/app/components/childrens-card/child-card/neurologists-examination/form.component.ts

```diff
@@ -0,0 +1,128 @@
+﻿import { Component, ElementRef, OnDestroy, ViewChild } from '@angular/core';
+import { NotificationsService, SimpleNotificationsComponent } from 'angular2-notifications';
+import { Subscription } from 'rxjs/Subscription';
+declare var $;
+
+import ChildCardModel from "../../../../models/child-card/child-card.model";
+import NeurologistsExaminationModel from "../../../../models/child-card/neurologists-examination/examination.model";
+
+import ChildCardService from '../../../../services/child-card.service';
+
+@Component({
+    moduleId: module.id,
+    providers: [
+        ChildCardService,
+        NotificationsService
+    ],
+    selector: 'neurologists-examination-form',
+    styleUrls: ['form.component.css'],
+    templateUrl: 'form.component.html'
+})
+export default class NeurologistsExaminationFormComponent implements OnDestroy {
+    private _childCard: ChildCardModel;
+    private _childCardSubscription: Subscription;
+    private _childCardService: ChildCardService;
+    private _isErrorOnLoading: boolean;
+    private _isErrorOnSaving: boolean;
+    private _isLoading: boolean;
+    private _isLoadingDiagnosis: boolean;
+    private _isSaving: boolean;
+    private _lastLoadingErrorMessage: string;
+    private _lastSavingErrorMessage: string;
+    private _neurologistsExamination: NeurologistsExaminationModel;
+    private _notificationOptions: any;
+    private _notificationService: NotificationsService;
+
+    constructor(childCardService: ChildCardService,
+        notificationService: NotificationsService) {
+        this._childCard = null;
+        this._childCardService = childCardService;
+        this._childCardSubscription = this._childCardService.currentChildCardObservable
+            .subscribe((childCard: ChildCardModel) => {
+                this._childCard = childCard;
+                this._loadExaminationFromServer();
+            });
+        this._isErrorOnLoading = this._isErrorOnSaving = false;
+        this._isLoading = true;
+        this._isSaving = false;
+        this._isLoadingDiagnosis = false;
+        this._lastLoadingErrorMessage = '';
+        this._lastSavingErrorMessage = '';
+        this._neurologistsExamination = new NeurologistsExaminationModel();
+        this._notificationOptions = {
```

Comment:
Consider using notificationOptions from shared service because this piece of code is redundant.